master/buildbot/reporters/mail.py

```diff
@@ -43,6 +43,11 @@
 from buildbot.reporters.message import MessageFormatter as DefaultMessageFormatter
 from buildbot.util import service
 
+if PY3:
```

Comment:
http://python-future.org/compatible_idioms.html#stringio

The recommendation of future author is to use

    from io import StringIO